Cesium.Runtime/StdIoFunctions.cs

```diff
@@ -332,7 +333,7 @@ public static int FPrintF(void* stream, byte* str, void* varargs)
                 case "f":
                     {
                         var floatNumber = ((double*)varargs)[consumedArgs];
-                        string floatNumberString = floatNumber.ToString("F" + (trim == -1 ? 6 : trim));
+                        string floatNumberString = floatNumber.ToString("F" + (trim == -1 ? 6 : trim), CultureInfo.InvariantCulture);
```

Comment:
Why have you added this here? Does it mess up with tests or something?

Anyway, I do not believe this is correct in a general sense.

This particular module is written to respect the current culture, keeping in mind that we'll integrate the C locale system with .NET's cultures eventually.

I would actually prefer to have invariant culture by default, but
- that's a discussion I'd like to separate from implementation if the unary plus operator
- if anything, then Cesium would have to set the current culture in `main()`

Reply:
Because without adding CultureInfo.InvariantCulture, Cesium.PrintF fails tests on non-InvariantCulture systems, which doesn't match the output in c.
![image](https://github.com/ForNeVeR/Cesium/assets/54708336/2423dd66-04ca-402e-9e54-ec38d413f56f)

Reply:
Ok, so all this business is incredibly wacky. I would still ask you to roll this change back, and let's discuss and reintroduce some proper mode in #560. This change should not indeed be related to the unary plus operator, and also to my conclusion in #560 it is also not the correct one.

Reply:
Done